scripts/encode_audio.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+OUTPUT_SAMPLE_RATE="8000"
+
+# directions:
+# make sure you run this script from the root directory e.g
+# scripts/conversion_script.sh
+
+input_dir="$(pwd)/assets/kyle_ai"
+# input_dir="$(pwd)/assets/silas_recorded"
+# input_dir="$(pwd)/assets/mp3"
+output_dir="$(pwd)/assets"
+
+
+# Ensure the output directory exists
+mkdir -p "$output_dir"
+
+# Remove all .wav files in the output directory
+rm -f "$output_dir"/*.wav
+
+# Loop through each mp3 file in the input directory
+# for file in "$input_dir"/*.mp3; do
+for file in "$input_dir"/two.wav  "$input_dir"/three.wav "$input_dir"/four.wav; do
```

Comment:
It looks like these are just doing the hard coded audio files? Where did all of the other ones go?